week-1/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,42 @@
+function percentage(Students, Mentors) {
+  var studentPercentage = Math.round(100 * Students / (Students + Mentors));
+  return studentPercentage;
+}
+
+function text(Students, Mentors) {
+  var findPercentage = percentage(Students, Mentors);
```

Comment:
Usually, variable names are nouns, and function names contain verbs - I think it's nice to represent them like this, given variables represent _things_ and functions _do_ things :)